src/fu-util.c

```diff
@@ -446,7 +446,8 @@ fu_util_build_device_tree (FuUtilPrivate *priv, GNode *root, GPtrArray *devs, Fw
 {
 	for (guint i = 0; i < devs->len; i++) {
 		FwupdDevice *dev_tmp = g_ptr_array_index (devs, i);
-		if (!(fwupd_device_has_flag (dev_tmp, FWUPD_DEVICE_FLAG_UPDATABLE) || priv->show_all_devices))
+		if (!fwupd_device_has_flag (dev_tmp, FWUPD_DEVICE_FLAG_UPDATABLE) &&
```

Comment:
Lost the `priv->show_all_devices`

Reply:
Also it seems that `fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE` should be in `fu_util_is_interesting_device`

Reply:
Gahh, copy paste issue. Will fix.